packages/components/src/button/docs/Button.stories.mdx

```diff
@@ -100,7 +122,7 @@ A button can vary in size.
 
 ### Disabled
 
-A button can be disabled.
+Used to disable a button that isn’t usable.
```

Comment:
Not sure about this change. The previous one seems better. I mean, disabling a button is always because he should not be usable.